src/ClientSymbols/QSettingsBasedStorageManagerTest.cpp

```diff
@@ -72,4 +73,20 @@ TEST_F(QSettingsBasedStorageManagerTest, LoadAndSaveModuleSymbolFileMappings) {
   EXPECT_THAT(loaded_mappings, testing::UnorderedElementsAreArray(mappings));
 }
 
+TEST_F(QSettingsBasedStorageManagerTest, LoadAndSaveDisabledModulePaths) {
+  absl::flat_hash_set<std::string> paths{"/path/to/module1", "/path/to/module2",
+                                         "/other/module/path"};
+
+  {
+    QSettingsBasedStorageManager manager;
+    EXPECT_TRUE(manager.LoadDisabledModulePaths().empty());
+    manager.SaveDisabledModulePaths(paths);
+  }
+
+  QSettingsBasedStorageManager manager;
+  absl::flat_hash_set<std::string> loaded_paths = manager.LoadDisabledModulePaths();
+
+  EXPECT_THAT(loaded_paths, testing::UnorderedElementsAreArray(paths));
+}
```

Comment:
Could you also delete all the paths again, to see if that works as well?

Reply:
done